docs/contributing/BuildFromSource.md

```diff
@@ -87,6 +81,17 @@ Executing `.\restore.cmd` or `.\build.cmd` may produce these errors:
 
 In most cases, this is because the option _Use previews of the .NET Core SDK_ in VS2019 is not checked. Start Visual Studio, go to _Tools > Options_ and check _Use previews of the .NET Core SDK_ under _Environment > Preview Features_.
 
+### Common error: path too long (on windows)
+If you encounter errors pointing to `path-too-long` errors, like:
+```
+Path: {{SOME-PATH}} exceeds the OS max path limit. The fully qualified file name must be less than 260 characters.
+```
+or
+```
+error LongPathsDisabled: Long paths are required for this project. See 'docs/contributing/LongPaths.md'.
+```
+see [LongPaths](LongPaths.md) on how to deal with this.
+
```

Comment:
I would remove this section. Instead I would add a line in the windows section above that Long path support is required and link to `Longpaths.md`

Reply:
agreed and done